features/step_definitions/pv.rb

```diff
@@ -164,14 +165,15 @@
     cache_resources *@result[:resource]
 
     # register mandatory clean-up
-    _pv = @result[:resource]
     _admin = admin
     teardown_add {
-      @result = _pv.delete(by: _admin)
```

Comment:
actually, you don't need to convert to `cli_exec`. You could just make this `_pv.delete(by: _admin, wait: "false")`.

Reply:
wait